src/aks-preview/azcli_aks_live_test/scripts/setup_venv.sh

```diff
@@ -106,6 +106,12 @@ setupAKSPreview(){
     source azEnv/bin/activate
 }
 
+createSSHKey(){
+    # create ssh-key in advance to avoid the race condition that is prone to occur when key creation is handled by
+    # azure-cli when performing test cases concurrently, this command will not overwrite the existing ssh-key
+    ssh-keygen -t rsa -b 2048 -C "azcli_aks_live_test@example.com" -f ~/.ssh/id_rsa -N "" -q <<< $'\n'
```

Comment:
can we do not use the ~/.ssh/id_rsa?

Reply:
sure, changed in commit [7afa50a](https://github.com/Azure/azure-cli-extensions/pull/3756/commits/7afa50ab215bee3290dfd2303d524b5bb35658d3).